controllers/articles.controller.js

```diff
@@ -9,3 +12,15 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticleById = (req, res, next) => {
+  articleId = req.params.article_id;
```

Comment:
You could save yourself a bit of legwork by de-structuring here.

Reply:
refactored, will show in next push